CONTRIBUTING.md

```diff
@@ -52,7 +52,7 @@ Here are a few examples:
 * New cop `ElseLayout` checks for odd arrangement of code in the `else` branch of a conditional expression. ([@bbatsov][])
 ```
 
-* Mark it up in [Markdown syntax][6].
+* Mark it up in [Markdown syntax][6] (in particular, ensure cop names are surrounded in backticks).
```

Comment:
Yeah, fair point. I wasn't sure if this change was worth it. I was mostly making it since I keep forgetting to do so on my PRs! I'll remove this commit since the other one is still valuable